packages/react-self-serializers/src/I18n.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useRef} from 'react';
+
+import {useSerialized, useHtmlAttributes} from '@shopify/react-html';
```

Comment:
what's going on with these added typescript files? do you just need to rebase?

Reply:
Yes, needed a rebase.